src/cmds/module/test.c

```diff
@@ -53,7 +53,9 @@ int main(int argc, char **argv) {
 	int test_nr = -1;
 	int opt;
 	/* TODO it must be agreed with shell maximum command length */
-	char test_name[100] = { 0 };
+	char test_name[100] = {
```

Comment:
This looks ugly. User should not be forced to change prior variation with this.
Are you sure there is no option to disable formatting structs (may be at all?)